core/modules/layout/includes/layout.layout.inc

```diff
@@ -168,6 +168,11 @@ function layout_layout_access_info() {
     'description' => t('Control access by the current path.'),
     'class' => 'PathLayoutAccess',
   );
+  $info['http_response_code'] = array(
+    'title' => t('Server response'),
+    'description' => t('Control access by the HTTP response status code being returned by the server.'),
```

Comment:
we could shorten this up a little bit by removing `being`: `Control access by the HTTP response status code returned by the server`